src/main/java/Duke.java

```diff
@@ -1,10 +1,160 @@
+import java.util.Scanner;
+
+/**
+ * Represents the bot Duke.
+ * Duke can perform tasks based on user input.
+ *
+ * {@value #NUM_OF_TASKS} Number of Tasks.
+ */
 public class Duke {
+
+    private static int NUM_OF_TASKS = 0;
+    private static Task[] list = new Task[100];
+
+    /**
+     * Runs other methods for main.
+     *
+     * @param args User Input.
+     */
     public static void main(String[] args) {
+        printGreetings();
+        listActions();
+    }
+
+    /**
+     * Prints greetings.
+     */
+    public static void printGreetings(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Yo Wassup, da name's\n" + logo);
+        System.out.println("Hi bro I'm Duke\n" + "Whatcha wanna do?");
+    }
+
+    /**
+     * Returns actions for lists based on user input.
+     */
+    public static void listActions() {
+        Scanner in = new Scanner(System.in);
+        String line;
+
+        do {
+            line = in.nextLine();
+            if (line.contains("done")){
+                updateTask(line,list);
+            }else if (line.equalsIgnoreCase("bye")){
+                printBye();
+                break;
+            }else if(line.equalsIgnoreCase("list")) {
+                printList(list);
+            }else if (line.contains("deadline")) {
+                createDeadline(line);
+                System.out.format("Now you have %s task%s in the list.\n", NUM_OF_TASKS,((NUM_OF_TASKS==1?"":"s")));
+            }else if (line.contains("event")) {
+                createEvent(line);
+                System.out.format("Now you have %s task%s in the list.\n", NUM_OF_TASKS,((NUM_OF_TASKS==1?"":"s")));
```

Comment:
Try to avoid complication expressions like "((NUM_OF_TASKS==1?"":"s"))", maybe use a String variable to store the result first, then use that variable for printing.